src/Console/ViewModelMakeCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Spatie\ViewModels\Console;
+
+use Illuminate\Console\GeneratorCommand;
+use Symfony\Component\Console\Input\InputOption;
+
+class ViewModelMakeCommand extends GeneratorCommand
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'make:view-model';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a new ViewModel class';
+
+    /**
+     * The type of class being generated.
+     *
+     * @var string
+     */
+    protected $type = 'ViewModel';
+
+    /**
+     * Execute the console command.
+     *
```

Comment:
Remove this docblock, also remove it for the other methods.

Reply:
done